emission/individual_tests/TestOverpass.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import unittest
+import os
+import requests
+import emission.net.ext_service.transit_matching.match_stops as enetm
+import logging
+
+#Set up query
+OVERPASS_KEY = os.environ.get("OVERPASS_KEY")
+
+#Sample loc1 = NREL East Gate
+loc1 = {'coordinates': [-105.16844103184974, 39.740428870224605]}
+#Sample loc2 = Denver Union Station
+loc2 = {'coordinates': [-105.00083982302972, 39.753710532185025]}
+#Sample loc3 = Grand Junction Train Station, CO
+loc3 = {'coordinates': [-108.57055213129632, 39.06472424640481]}
```

Comment:
It's great that you have identified these three locations, but I only see one of them being used, and it is hardcoded. Do you plan to use the other two?

Reply:
locs 2 and 3 are used in test_get_predicted_transit_mode, lines 54 and 55, passed in as stops 1 and 2